### DISCUSSION TITLE
Trouble with BLE32

### DISCUSSION THREAD

**Discussion by wjohn007** [arendst/Tasmota#22051] [Q&A]
I flashed a NodeMCU32 with Tasmota 14.2.0.3.
BLE was enabled.
MQTT server configured.
Telperiod 20.

The specific device A4C138CB0164 can be seen in the configuration screen.

![grafik](https://github.com/user-attachments/assets/f0249b60-5517-4948-9ee3-abfc52c82b71)

**However, it is not shown in the UI and does not exist in the sensor message**

tele/tasmota_Test02_635C38/SENSOR = 

It is reported in the BLE message
tele/tasmota_Test02_635C38/BLE 
```json
{
	"Time": "2024-08-29T17:03:13",
	"BLEDevices": {
		"total": 12,
		"A4C13886FFAA": {
			"i": 0,
			"r": -82
		},
		"A4C138CB0164": {
			"i": 1,
			"r": -72
		},
		"A4C138136501": {
			"i": 2,
			"r": -56
		},
		"001A221C375D": {
			"i": 3,
			"r": -72
		},
		"A4C138626BA5": {
			"i": 4,
			"r": -61
		},
		"E4AAEC9829D2": {
			"i": 5,
			"r": -82
		},
		"A4C138B39D8E": {
			"i": 6,
			"r": -87
		},
		"A4C138F2F740": {
			"i": 7,
			"r": -93
		},
		"A4C138E272DF": {
			"i": 8,
			"r": -92
		},
		"A4C138528576": {
			"i": 9,
			"r": -86
		},
		"492403040C60": {
			"i": 10,
			"r": -81
		},
		"A4C1386B1EA0": {
			"i": 11,
			"r": -92
		}
	}
}
```

What can be the reason for this behavior and how can I fix it?

Thank you for your support.
John

**Comment by sfromis** [comment]
Seeing the MAC does not tell much, and you did not mention what device it is. And if it is equivalent to your others.

**Comment by wjohn007** [comment]
It is a temperature and humidity sensor from XIAOMI called [LYWSD03MMC](https://www.mi.com/de/product/mi-temperature-and-humidity-monitor-2/specs/?srsltid=AfmBOopLMqKqTUM9SFIQNErFR1Yjev-B92D7tZ9j-YRkSUcJi4TQyYCk) which is identified by the first 6 digits of the MAC address “A4C138”.

**Reply by barbudor** [reply]
Are you running original firmware on the LYWSD03MMC ? Or have you flashed Pvvx firmware ? If so which version ? 

As of today Tasmota is supporting only the original xiaomi protocol and Pvvx custom protocol

I understand that recent versions of Pvvx firmware have dropped his own protocol in favor of BTHome which is not yet supported by Tasmota32-bluetooth

**Reply by wjohn007** [reply]
Thank you very much. That was a good tip.

Firmware used: Status: Hardware version: LYWSD03MMC B1.5, Software version: 4.7

The setting for Advertising-Type was wrong and should be:

![grafik](https://github.com/user-attachments/assets/a0a5d43e-033c-454d-9a9f-1271af95b60b)

Now everything works as expected.

John
